src/commands/utility/search-my-points.js

```diff
@@ -0,0 +1,77 @@
+const { SlashCommandBuilder } = require("discord.js");
+require("dotenv").config(); 
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("my-points-query")
+    .setDescription(
+      "Get the Discord search query to check points for a specific month"
+    )
+
+    .addIntegerOption((option) =>
+      option.setName("year").setDescription("Year to search").setRequired(false)
+    )
+
+    .addIntegerOption((option) =>
+      option
+        .setName("month")
+        .setDescription("Month to search")
+        .setRequired(false)
+    )
+
+    .addUserOption((option) =>
+      option
+        .setName("user")
+        .setDescription("User to search for")
+        .setRequired(false)
+    )
+
+    .addStringOption((option) =>
+      option
+        .setName("channel")
+        .setDescription("Select a specific channel")
+        .setRequired(false)
+        .addChoices(
+          { name: "novabot", value: "novabot" },
+          { name: "code-review", value: "code-review" }
+        )
```

Comment:
move this to a .env to make it easy to change

Reply:
I decided to delete them from env and it is better to enter them manually

![image](https://github.com/user-attachments/assets/19053024-c24c-4cb1-a62b-9b642f49f441)
![image](https://github.com/user-attachments/assets/b302a04f-9339-4ac2-909d-72fdcfbd4e95)

Reply:
Thanks!